tests/Request/RealtyRequestTest.php

```diff
@@ -168,6 +166,25 @@ public function testShortcutTypes()
         $this->assertEquals(['f' => ['type' => Type::RESIDENTIAL_SUBUNIT]], $request->getQuery());
     }
 
+    public function testBuildingProgress()
```

Comment:
Should not be neccessary. The test buildingProgress ist tested automatically, becaus it's defined in the FIELD constant.
You only have to add own tests for custom methods on the request (see testShortcutMarketingStates method in this class)